chain/README-dev.md

```diff
@@ -1,39 +1,114 @@
 # Developer Documentation
 
+## Branches
+
+We use `laika-node/pre-release`/`tlbc-node/pre-release` for release candidates
+and `laika-node/release`/`tlbc-node/release` for releases. We only merge from
+`master` to `*/pre-release`, from `*/pre-release` to `*/release` and from
+`*/release` back to `master`.
+
+The version on the master branch should always be a `dev` version. The
+`*/pre-release` branches should always have a `rc` version and on `*/release`
+branches we only have `prod` releases (where `prod` doesn't appear in the
+version number, i.e. the version number is always `major.minor.patch`).
+
+## bumpversion
+
+Both chains use [bumpversion](https://pypi.org/project/bumpversion/) in order to
+maintain their version. bumpversion can be used to automatically upgrade the
+current version in the respective `VERSION` file within the `chain/laika` and
+`chain/tlbc` directories.
+
+bumpversion is configured via the `.bumpversion.cfg` configuration files in
+`chain/laika` and `chain/tlbc`. If you want to run bumpversion to change
+a chain's version, please change to the chain's directory first. Note that
+bumping the version will automatically create a new commit.
+
+## Making A Pre-release
+
+Adopt the following instructions for the respective chain (demonstrated for
+the Laika test network) new version:
+
+```sh
+$ git checkout master
+$ git pull
+$ git checkout -b laika-node/prepare/pre-release-<version>
+$ cd chain/laika
+$ bumpversion release
+```
+
+and open a PR for this new branch to `laika-node/pre-release`.
+
+The version in the `*/pre-release` branches should always be a 'release candidate'
+version with the `rc` suffix.
+
+## Making A Release
+
+Run the following commands and adopt for the respective chain (demonstrated for
+the Laika test network) and new version:
+
+```sh
+$ git checkout -B laika-node/pre-release --track origin/laika-node/pre-release
```

Comment:
`git checkout laika-node/pre-release` should be enough, I dont know what you want with the other options

Reply:
If you have never done this you don't have such local branch. So this simply checks creates a local branch if it does not already exist, set the origin branch as upstream and switches to this (new) branch. Do you still want to simplify that?